fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -262,43 +311,120 @@ private fun TranslationsDialogInfoMessage(
     }
 }
 
+@Suppress("MagicNumber")
```

Comment:
Can we set the Magic Number used here into a variable to avoid this?

Reply:
Done